README.md

```diff
@@ -18,70 +18,53 @@ may well be used to fire requests to other services.
     ```bundle install```
 
 
-* How to run the test suite
-    * ```bundle exec rspec spec```
 
-* Services (job queues, cache servers, search engines, etc.)
-    
+## Initial setup
 
-* Deployment instructions
-    * check the code out and run ```rails s```
+Run ```make initial-setup```
 
-* play with application
-    Once the server is started you can actualy use postman to fire requests using the endpoints below
-    
+This will create the db schema  and run a  migration. you shouldn't have to run this again.
 
-* Developer local Endpoints
-    * ``http://localhost:3000/api/v1/applications``
-        * Only POST is supported at the moment so this wil create an application and retrun application ref
-    * ``http://localhost:3000/api/v1/status``
-        * Only GET is supported at the moment not sure anything else is needed here
+## Running the tests
 
+In order to run the tests run
 
-## Docker
+ * ```make test```
+ 
+ This is currently setup to run all rspec tests
 
-The docker file is created against a ruby alpine image using version 2.5.1 which is consistent
-with the latest version at the time of writing.
+## Running the application
 
-Alpine images are usually very lightweight and such are preferred choice for deploying containers.
+ * ```make serve```
```

Comment:
[Ditto](https://github.com/ministryofjustice/laa-apply-for-legalaid-api/pull/7/files#r210875636)